aries_vcx_core/src/anoncreds/credx_anoncreds.rs

```diff
@@ -28,14 +30,34 @@ use credx::{
     ursa::cl::MasterSecret as UrsaMasterSecret,
 };
 use indy_credx as credx;
+use serde::{de::DeserializeOwned, Deserialize, Serialize};
 use serde_json::Value;
 use uuid::Uuid;
 
 use super::base_anoncreds::BaseAnonCreds;
 
-const CATEGORY_CREDENTIAL: &str = "VCX_CREDENTIAL";
 const CATEGORY_LINK_SECRET: &str = "VCX_LINK_SECRET";
 
+const CATEGORY_CREDENTIAL: &str = "VCX_CREDENTIAL";
+const CATEGORY_CRED_DEF: &str = "VCX_CRED_DEF";
+const CATEGORY_CRED_KEY_PROOF: &str = "VCX_CRED_DEF_PROOF";
+const CATEGORY_CRED_DEF_PRIV: &str = "VCX_CRED_DEF_PRIV";
+const CATEGORY_CRED_SCHEMA: &str = "VCX_CRED_SCHEMA";
+const CATEGORY_CRED_SCHEMA_ID: &str = "VCX_CRED_SCHEMA_ID";
+
+const CATEGORY_REV_REG: &str = "VCX_REV_REG";
+const CATEGORY_REV_REG_DELTA: &str = "VCX_REV_REG_DELTA";
+const CATEGORY_REV_REG_INFO: &str = "VCX_REV_REG_INFO";
+const CATEGORY_REV_REG_DEF: &str = "VCX_REV_REG_DEF";
+const CATEGORY_REV_REG_DEF_PRIV: &str = "VCX_REV_REG_DEF_PRIV";
+
+#[derive(Debug, Deserialize, Serialize, Clone)]
+pub struct RevocationRegistryInfo {
+    pub id: RevocationRegistryId,
+    pub curr_id: u32,
+    pub used_ids: HashSet<u32>,
```

Comment:
I just copy pasted the info from `libvdrtools`. But yeah, these are better. However, this revocation info stuff is just a very bad way of tracking the current revocation ID. Right now we just replicate what `libvdrtools` was doing, but this will not make it in the refactor.

Reply:
I agree this would make migration a bit more difficult than it needs to be, as these structs are and were stored in wallet. if anything, we can do renaming in separate step.